app/views/qsets/_question_list_small_screen.html.erb

```diff
@@ -54,17 +54,6 @@
                 Another user
               <% end %>
             </div>
-            <div class="col-md-7">
```

Comment:
If you're removing this, looks like you can remove the unnecessary containing `.row` and `.col-md-5` nesting on lines 49-50 above. In other words, line 50 would just become a `.col-md-12`, and there's no point in that (unless we need an extra `.row`'s worth of padding).